app/src/scripts/dashboard/dashboard.searchresults.jsx

```diff
@@ -79,58 +122,57 @@ class SearchResults extends React.Component {
             loading: false,
           })
         })
+    } else {
+      this.setState({
+        results: '',
+        query: '',
+        loading: false,
+      })
     }
   }
 
-  render() {
-    let renderedResults = this.state.loading ? (
-      <Spinner active={true} />
-    ) : (
-      this._results(this.state.results)
-    )
-    return (
-      <div className="route-wrapper">
-        <div className="fade-in inner-route clearfix">
-          <div className="dashboard-dataset-teasers datasets datasets-private">
-            <div className="admin header-wrap clearfix">
-              <div className="row">
-                <div className="col-md-5">
-                  <h2>Search Results</h2>
-                </div>
-                <div className="col-md-7">
-                  <Search />
-                </div>
-              </div>
-              {renderedResults}
-            </div>
-          </div>
-        </div>
-      </div>
-    )
-  }
+  // Templating Methods ------------------------------------------------
 
   _results(results) {
     return results.map((result, index) => {
       let resultLink = this._resultLink(result)
       return (
         <div key={index} className="fade-in  panel panel-default">
-          <div className="panel-heading">
-            <div className="header clearfix">{resultLink}</div>
+          <div className="panel-body">
+            <div className="panel-heading">
+              <div className="header clearfix">{resultLink}</div>
+            </div>
           </div>
         </div>
       )
     })
   }
 
   _resultLink(result) {
+    let type
+    let arr = result.link.split('/')
+    if (arr.includes('file-display')) {
+      type = 'File Display - ' + arr[8]
+    } else if (arr.includes('versions')) {
+      type = 'Dataset - Snapshot'
+    } else if (!arr.includes('public') && arr.includes('datasets')) {
+      type = 'Dataset - Draft'
+    } else if (arr.includes('openneuro.org') && !arr.includes('public')) {
+      type = 'Page - Home'
+    } else {
+      type = 'Page'
+    }
+
```

Comment:
This part seems a bit fragile as it will break as pages change. I think it would be better to take this out and improve the titles for the pages indexed, which would improve the search visibility with or without viewing them via OpenNeuro.

Reply:
Ah. Indeed. I was doing the best that I knew how with the limited amount of information being returned from the API. Should I reach out to Ross about updating the titles and possibly adding a type property? I tried it without the type, and the results are very confusing. There isn't a way to tell the difference between a dataset draft, a snapshot, or a file display.

Reply:
#85 is part of the issue since some pages are orphaned by it and the index is updated slowly or not at all. Mostly the titles for pages just don't include useful information and I think that's a separate issue, I filed #529 for that.

Reply:
- [x] removed